.gitignore

```diff
@@ -9,6 +9,7 @@
 /web/package-lock.json
 /dist/*.bin
 /dist/docs
+/dist/*.h
```

Comment:
This should actually not be in `.gitignore`. It's a little ugly, but I intend to check in the compiled webpage so that people don't need to have a webdev environment set up in order to build the project (the build script falls back on using the existing `.h`).